### ISSUE TITLE
Fix the Gitconfig Provider for Gitpod

### ISSUE THREAD

**Issue by PrashamTrivedi** [PrashamTrivedi/gitconfig-provider#4]
In Gitpod running gitconfig provider provides an error. 

```
gitconfig-provider: 1: Syntax error: end of file unexpected (expecting ")")
```

**Comment by PrashamTrivedi** [comment]
One more report. Providers should be case insensitive

**Comment by PrashamTrivedi** [comment]
Fixed in 3937114829132901f0c2d293dac0ca6fdfa9f0d0
